src/components/ListComposable/components/ListItemRenderer/ListItemRenderer.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable react/display-name */
+import React from 'react';
+
+import {ChevronDown, ChevronUp} from '@gravity-ui/icons';
+
+import {Icon} from '../../../Icon';
+import {Label} from '../../../Label';
+import {Text} from '../../../Text';
+import type {
+    ListItemBaseData,
+    ListItemRendererProps as ListItemRendererPropsBase,
+} from '../../types';
+import {createListItemQa} from '../../utils/createListItemQa';
+import {useListContext} from '../ListContext/ListContext';
+import {ListItemView} from '../ListItemView/ListItemView';
+
+interface ListItemRendererProps extends ListItemRendererPropsBase<ListItemBaseData> {}
+
+export const ListItemRenderer = React.memo(({item, index}: ListItemRendererProps) => {
```

Comment:
U can add name to this function and remove `/* eslint-disable react/display-name */`